tools/js-sdk-release-tools/src/test/pipeline/sdkGeneration.test.ts

```diff
@@ -44,6 +44,7 @@ describe('Auto SDK generation path test', () => {
         const fakeTspConfig = {
             options: {
                 '@azure-tools/typespec-ts': {
+                    'is-modular-library': true,
```

Comment:
add more tests: 
1. only contains `isModularLibrary`
2. only contains `is-modular-library` 
3. contains both